src/ansys/mechanical/core/embedding/app.py

```diff
@@ -201,7 +203,22 @@ def exit(self):
         else:
             self.ExtAPI.Application.Exit()
 
-    def execute_script(self, script: str):
+    def update_globals(
+        self, globals_dict: typing.Dict[str, typing.Any], enums: bool = True
+    ) -> None:
+        """Use to update globals variables.
+
+        When scripting inside Mechanical, the Mechanical UI will automatically
+        set global variables in python. PyMechanical can not do that automatically,
+        but this method can be used.
+        `app.update_globals(globals())`
+
+        By default, all enums will be imported too. To avoid including enums, set
+        the `enums` argument to False.
```

Comment:
By default, all enums will also be imported. To not import enums, set the `enums` argument to False.